python/tests/test_tables.py

```diff
@@ -112,6 +112,14 @@ def test_low_level_get_row(self):
         with pytest.raises(_tskit.LibraryError):
             t.ll_table.get_row(1)
 
+    def test_low_level_equivalent(self):
```

Comment:
This test is looking for an `equivalent` method on each individual table, which I don't think you were planning to implement.